Maskbook/Scene/App/FileService/View/TappableText.swift

```diff
@@ -27,7 +27,8 @@ struct TappableText: UIViewRepresentable {
     }
 
     func updateUIView(_ uiView: UITextView, context: Context) {
-        Task(priority: .high) { @MainActor in
+        DispatchQueue.main.async {
```

Comment:
Can you add some comments here on why we need to use  `DispatchQueue.main.async`?